bottle_server.py

```diff
@@ -41,7 +41,7 @@
 # import pg_logger
 import urllib
 import urllib2
-
+import MetadataJsonPrinter
```

Comment:
`MetadataJsonPrinter` is a module instead of a class. Thus change name from `MetadataJsonPrinter` to `metadata_json_printer`.